CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 4.2.0 (YYYY-MM-DD)
```

Comment:
4.2.0 entry need to be removed since this is going to be merged to the `releases` branch.